lib/web/JsonpMainTemplatePlugin.js

```diff
@@ -121,15 +121,17 @@ class JsonpMainTemplatePlugin {
 						: "",
 					"script.charset = 'utf-8';",
 					`script.timeout = ${chunkLoadTimeout / 1000};`,
+					`script.src = ${mainTemplate.requireFn}.p + ${scriptSrcPath};`,
```

Comment:
Please move setting `src` back to the end. Some versions of IE, start downloading once `src` is set.

You can create a temp variable to store the src url and assign it in the end to be able to use it for crossOriginLoading.

Reply:
Unfortunately, the elegance of this implementation relies on the fact that setting script src produces an absolute URL, even if the original path is relative. Which versions of IE suffer from that problem? I think not all version of IE support the `crossorigin` attribute anyways, it might be possible to keep setting `src` at the end, but move setting the `crossorigin` attribute after.

The alternative might be producing an absolute URL via something like: https://stackoverflow.com/a/24437713